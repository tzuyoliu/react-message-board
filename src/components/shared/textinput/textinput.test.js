import React from 'react'
import { create } from 'react-test-renderer'
import TextInput from '~/components/shared/textinput'

describe('textinput.test.js', () => {
    test('<TextInput /> should render', () => {
        const textinput = create(<TextInput />)
        expect(textinput).toBeTruthy()
    })

    test('<TextInput /> should match the snapshot', () => {
        const textinput = create(<TextInput />)
        expect(textinput.toJSON()).toMatchSnapshot()
    })
})
