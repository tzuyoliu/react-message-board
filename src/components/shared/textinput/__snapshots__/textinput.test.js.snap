// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`textinput.test.js <TextInput /> should match the snapshot 1`] = `
<input
  className="sc-bdVaJa gjZuiT"
/>
`;
